.github/workflows/build_and_push_api_image.yml

```diff
@@ -0,0 +1,73 @@
+name: "Build Moped API image and push to AWS ECR"
+on:
+  push:
+    # branches:
+    # - main
+    # - production
```

Comment:
This gets commented back in right before merger. Leaving it out allows the CI to be tested as part of this PR.